docs/WEBPACK.md

```diff
@@ -83,3 +83,50 @@ if (ENV === "development") {
 	/* the code inside this block will be executed only in development */
 }
 ```
+
+
+#### [HtmlElementWebpackPlugin](https://www.npmjs.com/package/html-elements-webpack-plugin "HtmlElementWebpackPlugin")
+
+This plugin allows the webpack plugin to append head elements during the creation of index.html. 
```

Comment:
I think there is a typo error here.
Maybe "This (webpack) plugin appends head elements ..." or something like this would be better 😊

Reply:
Done!